components/builder-depot/src/server.rs

```diff
@@ -2311,13 +2540,8 @@ where
         download_origin_key,
         "origin_key",
     );
-    r.post(
-        "/origins/:origin/encryption_keys",
-        XHandler::new(generate_origin_encryption_keys).before(basic.clone()),
-        "origin_encryption_key_generate",
-    );
     r.get(
-        "/origins/:origin/encryption_keys/latest",
+        "/origins/:origin/encryption_key",
```

Comment:
It might be worth adding the `breaking-changes` label (do we have one yet?) because of this part - just for the sake of release notes.  I think the odds of anyone using our HTTP API directly (independent of the `hab` CLI) are low, but just in case, it's worth pointing out that this is a breaking change.

Reply:
good point. I thought it was safe since we hadn't shipped to prod yet, but we did that yesterday